packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -399,62 +398,67 @@ function DragIntoListExample() {
 
   // Append a generated key to the item type so they can only be dragged between these two lists
   let dragType = React.useMemo(() => `keys-${Math.random().toString(36).slice(2)}`, []);
-  let acceptedDragTypes = `${dragType}-file`;
-
-  // Update the target and source list upon successful drop
-  let onMove = (keys, target) => {
-    if (target.type === 'root') {
-      targetList.append(...keys.map(key => sourceList.getItem(key)));
-    } else {
-      switch (target.dropPosition) {
-        case 'before':
-          targetList.insertBefore(target.key, ...keys.map(key => sourceList.getItem(key)));
-          break;
-        case 'after':
-          targetList.insertAfter(target.key, ...keys.map(key => sourceList.getItem(key)));
+  let acceptedDragTypes = [`${dragType}-file`];
```

Comment:
TODO: the docs locally were behaving kinda buggy (drop indicators not showing up/lagging/arrow key nav between drop targets weirdness) but the same examples worked just fine in the storybook when pasted there. Need to investigate

Reply:
Reverted to before the DnD api simplification was merged into main, issue still persists in the docs. Did some more digging and it looks like somehow 2072a5a949768f56d5b2eee50cae667404c6f771 broke the indicators in the docs but not in storybook??